### PR TITLE
[FEATURE] AbsoluteQuantitationMethodFile::store()

### PR DESCRIPTION
**Pull request by pasqoo**
Implementation proposal.

Given a filename in input and a vector of `AbsoluteQuantitationMethod`, store the data to a .csv file.

### CODE REVIEW

File: src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp [hunk 1]
```diff
@@ -261,6 +261,34 @@ START_SECTION((void load(const String & filename, std::vector<AbsoluteQuantitati
 
 END_SECTION
 
+START_SECTION(void store(const String & filename, const std::vector<AbsoluteQuantitationMethod> & aqm_list) const)
+  AbsoluteQuantitationMethodFile aqmf;
+  vector<AbsoluteQuantitationMethod> aqm_list1, aqm_list2;
+  aqmf.load(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv"), aqm_list1);
+  aqmf.store(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_2.csv"), aqm_list1);
+  aqmf.load(OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv"), aqm_list2);
```
Comment (hroest) [inline src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp]: shouldnt this load ` "AbsoluteQuantitationMethodFile_2.csv"` in order to really test the store method?

File: src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp [hunk 2]
```diff
@@ -261,6 +261,34 @@ START_SECTION((void load(const String & filename, std::vector<AbsoluteQuantitati
...
+  TEST_EQUAL(aqm_list1.size(), aqm_list2.size())
+  for (Size i = 0; i < aqm_list1.size(); ++i)
```
Comment (hroest) [inline src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp]: if `AbsoluteQuantitationMethod` had an `operator==` then you could just test for equality. But this is fine too...

File: src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp (hunk 1 above)
Comment (pasqoo) [reply src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp]: Yeah, of course it should! Thanks for catching this. During the tests I kept looking at the files outputs and they were identical except for the end of lines (no '\r' in the output files). Didn't notice this bug.

**Comment by hroest** [comment]
looks good (only failed tests are crux adapter)

File: src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp (hunk 2 above)
Comment (pasqoo) [reply src/tests/class_tests/openms/source/AbsoluteQuantitationMethodFile_test.cpp]: I will implement this in the coming days

**Pull request merged**